cmd/csv-merger/csv-merger.go

```diff
@@ -140,6 +140,32 @@ func main() {
 				}
 
 				installStrategyBase.DeploymentSpecs = append(installStrategyBase.DeploymentSpecs, csvStruct.Spec.InstallStrategy.StrategySpec.DeploymentSpecs...)
+
+				// We need to add the "env" section from this Service Operator deployment in case there
+				// are default values configured there that are needed for use with defaulting webhooks
+				//
+				// - DeploymentSpecs[0] is always the base deployment for OpenStack Operator
+				// - Container at index 1 in DeploymentSpecs[0].Spec.Template.Spec.Containers list is
+				//   always the OpenStack Operator controller-manager
+				// - We need to find the Service Operator's controller-manager container in
+				//   csvStruct.Spec.InstallStrategy.StrategySpec.DeploymentSpecs[0].Spec.Template.Spec.Containers
+				//
+				// TODO: What about "env" list keys that overlap between service operators (i.e. non-unique
+				//       names)?
+
+				for _, container := range csvStruct.Spec.InstallStrategy.StrategySpec.DeploymentSpecs[0].Spec.Template.Spec.Containers {
+					if container.Name == "manager" {
+						installStrategyBase.DeploymentSpecs[0].Spec.Template.Spec.Containers[1].Env = append(
+							// OpenStack Operator controller-manager container env vars
+							installStrategyBase.DeploymentSpecs[0].Spec.Template.Spec.Containers[1].Env,
+							// Service Operator controller-manager container env vars
+							container.Env...,
+						)
+						// We found the controller-manager container, so no need to continue iterating
+						break
+					}
+				}
```

Comment:
The above part looks good. In addition to that however don't we also need to append an extra ENV to all service operators (glance-operator, etc.) so that we set ENABLE_WEBHOOKS=false for each as when they are installed alongside of openstack-operator we don't want the local webhooks enabled there.

Reply:
Good call.  I will add that.

Reply:
@dprince Should we also remove the `webhookDefinitions` incorporation?  If we're setting `ENABLE_WEBHOOKS=false` in the service operator's deployment's env vars, then we wouldn't want there to be `ValidatingWebhookConfiguration`s or `MutatingWebhookConfiguration`s defined for it.  Otherwise those will still try to call the webhook server (which is not running) and throw an error whenever someone creates/update the service operator's CR(s).

Reply:
Yes, sounds good. lets remove them